### DISCUSSION TITLE
EOM alignment problem with protein_RNA

### DISCUSSION THREAD

**Discussion by franke-biosaxs** [biosaxs-com/atsas-community#1425] [Mixtures and flexible systems]
On 2016-01-10 20:37:31, user Charles wrote:

Dear all,

I am trying to use eom in order to fit our SAXS data for a protein_RNA complex (1:1) with the individual X-ray structures (1 protein pdb and 1 RNA pdb).
I have tried several combinations however I always get the message "alignment problems" (see .txt).
Thinking that this might be coming from the RNA pdb (eom runs properly with the protein) I have tried modifying it with no success (initial.pdb and test.pdb for further details).
Any suggestions would be great! What am I missing?

Thank you all
Cheers

Attachments:

  - [28664_89ad468f7d2cff1a8537fbe870d30762.pdb](https://github.com/biosaxs-com/atsas-community/blob/saxier-org/forum/attachments/28664_89ad468f7d2cff1a8537fbe870d30762.pdb)
  - [28664_58d590894dc964fff3f511a04b84135d.txt](https://github.com/biosaxs-com/atsas-community/blob/saxier-org/forum/attachments/28664_58d590894dc964fff3f511a04b84135d.txt)
  - [28664_a996cd148e42f00c3731597cec3375ec.pdb](https://github.com/biosaxs-com/atsas-community/blob/saxier-org/forum/attachments/28664_a996cd148e42f00c3731597cec3375ec.pdb)
  - [28664_ad9c6344e729dd851386b566deb9dcd3.seq](https://github.com/biosaxs-com/atsas-community/blob/saxier-org/forum/attachments/28664_ad9c6344e729dd851386b566deb9dcd3.seq)

**Comment by franke-biosaxs** [comment]
On 2016-01-11 12:58:29, user Alex wrote:

maybe if you change everything in RNA file to HETATM instead of ATOM, it will work.
It worked for me before (i assume u use latest release - I tried older version).

note, however, that you might have some clashes in ensemble between protein and RNA depending where
modeled part is.

HTH, Alex

**Comment by franke-biosaxs** [comment]
On 2016-01-15 16:13:44, user Charles wrote:

Indeed I use the latest version.
I have tried your suggestion however I still get the same error!
There are no modelled parts (neither for the protein nor the RNA) since all the residues are observed in the X-ray structures.
Clashes are unlikely since the SAXS envelopes of the separate macromolecules fit well with the one of the complex; also sasref was able to build a model chi*2 of 1.4
Most probably the problem is related to the RNA.pdb?

Further advices would be appreciated
Thanks

Charles

**Comment by franke-biosaxs** [comment]
On 2016-01-15 16:24:09, user Alex wrote:

> There are no modelled parts (neither for the protein nor the RNA) since all the residues are observed in the X-ray structures.

i don't understand what you are trying to do with EOM then?! I initially thought that some part in your protein is missing and you were trying
to sample the conformation of a missing part in a complex with RNA.

If your structures are complete, you just need to validate complex using CRYSOL or SASREF (if orientation of protein to RNA is not known).

HTH, Alex

**Comment by franke-biosaxs** [comment]
On 2016-01-15 17:45:54, user Charles wrote:

The thing is that the model obtained from sasref shows the protein rotated on the x axis by 180° compared to structural alignments with known homologue proteins complexed with RNA, which is quite surprising for such a conserved domain.
So I was curious to see if eom (that can handle RNA in contrast to bunch, as far as I know) could generate a model similar to the one from sasref or not...
Basically I am trying to model this protein-RNA complex with different suites to see if they do converge or not.
I do realize that I can add constraints or rotations in sasref but I do not know if my above arguments are enough to justify it!

Charles

**Comment by franke-biosaxs** [comment]
On 2016-01-15 19:36:00, user Alex wrote:

Charles,

SASREF is meant to work with complete structures. It looks for optimal translation and orientation of the subunits in the complex based on SAXS data.
I also suggest that you start your SASREF with protein and RNA shifted to the center of the coordinates and protein subunit fixed. Start it 10 times to see
whether you get converging solution. This way SASREF will look for such complex arrangement that would satisfy scattering data, i.e. discrepancy between theoretical curve
from your model will be minimal with experimental data.

BUNCH is used for modeling single chain proteins with missing linkers. CORAL is a mash-up of BUNCH and SASREF, supporting multichain assemblies.

EOM models single chain proteins (including symmetric objects if you know oligomerization interface) by sampling conformation of the missing part without SAXS data.
On the second step, it selects ensembles based on saxs data.  In your case, it would only make sense to use EOM if smth was missing in your protein. In other words, EOM will have two of your subunits fixed and would not do anything.

HTH, Alex
